include/rdma/fi_eq.h

```diff
@@ -75,10 +77,18 @@ struct fid_wait {
 	struct fi_ops_wait	*ops;
 };
 
+#ifndef _WIN32
 struct fi_mutex_cond {
 	pthread_mutex_t		*mutex;
 	pthread_cond_t		*cond;
 };
+#else /* _WIN32 */
+struct fi_critsec_cond {
+	CRITICAL_SECTION	*critsec;
+	CONDITION_VARIABLE	*cond;
+};
+#endif /* _WIN32 */
```

Comment:
What you have so far fi_eq.h is fine.  Just add a new enum value to
```c
enum fi_wait_obj {
    ...
    FI_WAIT_CRITSEC_COND,
};
```
Both the new structure and new enum value need to be documented in the man pages.